src/main.js

```diff
@@ -3,13 +3,19 @@ const { setup } = require('./library.js');
 const { createDeployment } = require('./deployments.js');
 
 async function run(context) {
-  await createDeployment(context);
+  return await createDeployment(context);
 }
 
 try {
   const setupContext = setup();
   const runPromise = new Promise((resolve, reject) => resolve(run(setupContext)));
-  runPromise.then(deploymentId => core.setOutput('github-deployment-id', deploymentId));
+  runPromise.then(deploymentId => {
+    // The token can be sent by and filtered out in the log
+    // but we'll delete it just to be safe.
+    delete setupContext.token;
+    console.log(`Created deployment ${deploymentId}:`, setupContext);
```

Comment:
This has me feeling uneasy... I'm not so concerned about the token. That's addressed in your comment. I'm concerned about the potential "noise" in the log. that setupContext object has attributes like `server_url`, `workflow_run_id`, `owner`, and `repo` that aren't useful in the log at this point. Is there a better way to collect just the action's _inputs_ and report them in the log?

Also, if we're going to log the inputs, they'd be just as useful on a failure as on a success. But I forget, does the run log already show us actions inputs?

Reply:
This was in here for confirmation that what in is what was run all the way through.  I don't have a problem with taking it out.

The logged inputs should be enough of a means of debugging at this point.